packages/block-library/src/navigation/block.json

```diff
@@ -50,13 +50,11 @@
 		"showSubmenuIcon": "showSubmenuIcon"
 	},
 	"supports": {
```

Comment:
Should we remove the fontSize attribute from block.json as we did for paragraph?